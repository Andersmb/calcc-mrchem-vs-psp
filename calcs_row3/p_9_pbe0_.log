Starting job 5688133 on c2-[33,38-39],c3-6,c5-40 at Fri May 20 06:07:16 CEST 2022

Copying p_9_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_9_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688133       p_9_pbe0_        180                                             01:35:49      0:0 
5688133.bat+      batch         36        1   00:00:04          0   00:00:04   01:35:49      0:0 
5688133.ext+     extern        180        5   00:00:00          3   00:00:00   01:35:49      0:0 
5688133.0      mrchem.x        180       30   01:34:42         29   04:22:05   01:34:53      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688133                                                                          
5688133.bat+     19280K          0     19280K        0              0          0 
5688133.ext+          0          3          0        0              3          0 
5688133.0     32317104K         29 109153155+        0              2          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688133                                                                                               
5688133.bat+       19.42M               0         19.42M        0.07M                0          0.07M 
5688133.ext+        0.00M               3          0.00M            0                3              0 
5688133.0           6.75M               0          6.74M      266.98M                0        165.23M 

Job 5688133 completed at Fri May 20 07:43:08 CEST 2022
