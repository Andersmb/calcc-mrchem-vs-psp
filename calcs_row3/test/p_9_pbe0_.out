                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :       -0.000000        0.000000        0.000000
    1       F            :        2.898840        0.000000        0.000000
    2       F            :       -1.449420        2.510501        0.000000
    3       F            :       -1.449420       -2.510501        0.000000
    4       F            :       -0.000000        0.000000        2.980098
    5       F            :       -0.000000        0.000000       -2.980098
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              30
 Beta electrons          :                                              30
 Total electrons         :                                              60
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999754063e-01
    1    1    a          :                              9.999998736407e-01
    2    1    a          :                              9.999998780002e-01
    3    1    a          :                              9.999993377532e-01
    4    1    a          :                              1.000000321522e+00
    5    1    a          :                              1.000000004121e+00
    6    1    a          :                              9.999999581126e-01
    7    1    a          :                              1.000000000854e+00
    8    1    a          :                              9.999999987350e-01
    9    1    a          :                              9.999999880806e-01
   10    1    a          :                              9.999993654141e-01
   11    1    a          :                              9.999999969128e-01
   12    1    a          :                              9.999999434437e-01
   13    1    a          :                              9.999998308908e-01
   14    1    a          :                              1.000001526489e+00
   15    1    a          :                              1.000001278059e+00
   16    1    a          :                              9.999999376986e-01
   17    1    a          :                              9.999998264044e-01
   18    1    a          :                              1.000000071428e+00
   19    1    a          :                              1.000000197643e+00
   20    1    a          :                              9.999995365259e-01
   21    1    a          :                              1.000000322022e+00
   22    1    a          :                              1.000000179883e+00
   23    1    a          :                              1.000000110542e+00
   24    1    a          :                              1.000000309079e+00
   25    1    a          :                              9.999999713895e-01
   26    1    a          :                              9.999999411095e-01
   27    1    a          :                              9.999998327712e-01
   28    1    a          :                              1.000000072346e+00
   29    1    a          :                              9.999871427671e-01
   30    1    b          :                              9.999999754063e-01
   31    1    b          :                              9.999998728777e-01
   32    1    b          :                              9.999998750421e-01
   33    1    b          :                              9.999993508117e-01
   34    1    b          :                              1.000000234295e+00
   35    1    b          :                              9.999999922614e-01
   36    1    b          :                              9.999999679183e-01
   37    1    b          :                              1.000000001264e+00
   38    1    b          :                              9.999999936207e-01
   39    1    b          :                              9.999999880809e-01
   40    1    b          :                              9.999993654139e-01
   41    1    b          :                              9.999999966140e-01
   42    1    b          :                              9.999999361940e-01
   43    1    b          :                              9.999998349411e-01
   44    1    b          :                              1.000001489381e+00
   45    1    b          :                              1.000001278065e+00
   46    1    b          :                              9.999999394430e-01
   47    1    b          :                              9.999998264044e-01
   48    1    b          :                              1.000000065357e+00
   49    1    b          :                              1.000000193691e+00
   50    1    b          :                              9.999995354996e-01
   51    1    b          :                              1.000000315443e+00
   52    1    b          :                              1.000000179883e+00
   53    1    b          :                              1.000000099704e+00
   54    1    b          :                              1.000000309079e+00
   55    1    b          :                              9.999999713895e-01
   56    1    b          :                              9.999999411095e-01
   57    1    b          :                              9.999998327712e-01
   58    1    b          :                              1.000000072346e+00
   59    1    b          :                              9.999871431099e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                832.058160404969
 E-N energy              :            (au)              -2806.540015514627
 Coulomb energy          :            (au)                805.394742626605
 Exchange energy         :            (au)                -18.377503485956
 X-C energy              :            (au)                -57.108967497394
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                409.197084707916
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1244.573583466403
 Nuclear energy          :            (au)                409.197084707916
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.353764987585e+02
                         :      (kcal/mol)             -5.242066631510e+05
                         :        (kJ/mol)             -2.193280680047e+06
                         :            (eV)             -2.273175156944e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.369117620321
    1    1    a          :            (au)                -24.626482760980
    2    1    a          :            (au)                -24.625472691290
    3    1    a          :            (au)                -24.624695378280
    4    1    a          :            (au)                -24.478114477574
    5    1    a          :            (au)                -24.478028715317
    6    1    a          :            (au)                 -7.144314438534
    7    1    a          :            (au)                 -5.289042978598
    8    1    a          :            (au)                 -5.288978365601
    9    1    a          :            (au)                 -5.286856083553
   10    1    a          :            (au)                 -1.411482400001
   11    1    a          :            (au)                 -1.321979891841
   12    1    a          :            (au)                 -1.321462618014
   13    1    a          :            (au)                 -1.237375994632
   14    1    a          :            (au)                 -1.191607148293
   15    1    a          :            (au)                 -0.714903963722
   16    1    a          :            (au)                 -0.622543333847
   17    1    a          :            (au)                 -0.621845034203
   18    1    a          :            (au)                 -0.575833266705
   19    1    a          :            (au)                 -0.502759945565
   20    1    a          :            (au)                 -0.502513557352
   21    1    a          :            (au)                 -0.502039077745
   22    1    a          :            (au)                 -0.501768037206
   23    1    a          :            (au)                 -0.468692397793
   24    1    a          :            (au)                 -0.430798861225
   25    1    a          :            (au)                 -0.421413150571
   26    1    a          :            (au)                 -0.384816435486
   27    1    a          :            (au)                 -0.384583297378
   28    1    a          :            (au)                 -0.361992137355
   29    1    a          :            (au)                 -0.361867141415
   30    1    b          :            (au)                -77.369118768436
   31    1    b          :            (au)                -24.626482864520
   32    1    b          :            (au)                -24.625473489699
   33    1    b          :            (au)                -24.624696017671
   34    1    b          :            (au)                -24.478114196664
   35    1    b          :            (au)                -24.478028872248
   36    1    b          :            (au)                 -7.144313658073
   37    1    b          :            (au)                 -5.289043868570
   38    1    b          :            (au)                 -5.288976846138
   39    1    b          :            (au)                 -5.286855694905
   40    1    b          :            (au)                 -1.411505448269
   41    1    b          :            (au)                 -1.321976970631
   42    1    b          :            (au)                 -1.321462609809
   43    1    b          :            (au)                 -1.237376618515
   44    1    b          :            (au)                 -1.191595653774
   45    1    b          :            (au)                 -0.714895842241
   46    1    b          :            (au)                 -0.622543558300
   47    1    b          :            (au)                 -0.621839452502
   48    1    b          :            (au)                 -0.575832394308
   49    1    b          :            (au)                 -0.502759211412
   50    1    b          :            (au)                 -0.502508072737
   51    1    b          :            (au)                 -0.502042475432
   52    1    b          :            (au)                 -0.501764140493
   53    1    b          :            (au)                 -0.468690579035
   54    1    b          :            (au)                 -0.430797437373
   55    1    b          :            (au)                 -0.421412538026
   56    1    b          :            (au)                 -0.384811403593
   57    1    b          :            (au)                 -0.384581320968
   58    1    b          :            (au)                 -0.361993946089
   59    1    b          :            (au)                 -0.361866986753
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -474.106742137581
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          7.745967e+00        -835.376498758487         -8.353765e+02
    1          4.993907e-01        -840.159814606776         -4.783316e+00
    2          1.971869e-01        -840.364291949962         -2.044773e-01
    3          1.113041e-01        -840.402588844123         -3.829689e-02
    4          5.792406e-02        -840.414419691747         -1.183085e-02
    5          5.731050e-02        -840.423641415557         -9.221724e-03
    6          1.835313e-02        -840.424720921835         -1.079506e-03
    7          1.856330e-02        -840.425418569529         -6.976477e-04
    8          6.303594e-03        -840.425499075096         -8.050557e-05
    9          5.041437e-03        -840.425541758471         -4.268338e-05
   10          2.110729e-03        -840.425552878277         -1.111981e-05
   11          2.074413e-03        -840.425559563334         -6.685057e-06
   12          8.589430e-04        -840.425561035547         -1.472213e-06
   13          5.638548e-04        -840.425561737723         -7.021763e-07
   14          3.904763e-04        -840.425562033997         -2.962740e-07
   15          2.195532e-04        -840.425562141971         -1.079734e-07
   16          1.005493e-04        -840.425562172364         -3.039372e-08
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :       -0.000000        0.000000        0.000000
    1       F            :        2.898840        0.000000        0.000000
    2       F            :       -1.449420        2.510501        0.000000
    3       F            :       -1.449420       -2.510501        0.000000
    4       F            :       -0.000000        0.000000        2.980098
    5       F            :       -0.000000        0.000000       -2.980098
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                838.594668943046
 E-N energy              :            (au)              -2821.075814633522
 Coulomb energy          :            (au)                808.156601350869
 Exchange energy         :            (au)                -18.302670726948
 X-C energy              :            (au)                -56.995431813726
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                409.197084707916
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1249.622646880280
 Nuclear energy          :            (au)                409.197084707916
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.404255621724e+02
                         :      (kcal/mol)             -5.273749982528e+05
                         :        (kJ/mol)             -2.206536994122e+06
                         :            (eV)             -2.286914357813e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.468205683888
    1    1    a          :            (au)                -24.841701952306
    2    1    a          :            (au)                -24.841698609874
    3    1    a          :            (au)                -24.841694023513
    4    1    a          :            (au)                -24.797497955758
    5    1    a          :            (au)                -24.797497081889
    6    1    a          :            (au)                 -6.836474532638
    7    1    a          :            (au)                 -4.975492260480
    8    1    a          :            (au)                 -4.975491915130
    9    1    a          :            (au)                 -4.972617019994
   10    1    a          :            (au)                 -1.379766207892
   11    1    a          :            (au)                 -1.316232062180
   12    1    a          :            (au)                 -1.316227606577
   13    1    a          :            (au)                 -1.277214549482
   14    1    a          :            (au)                 -1.245392474356
   15    1    a          :            (au)                 -0.741021069322
   16    1    a          :            (au)                 -0.642524945297
   17    1    a          :            (au)                 -0.642523532616
   18    1    a          :            (au)                 -0.628911140568
   19    1    a          :            (au)                 -0.571288517924
   20    1    a          :            (au)                 -0.571286086098
   21    1    a          :            (au)                 -0.549938517993
   22    1    a          :            (au)                 -0.549935619292
   23    1    a          :            (au)                 -0.524155445170
   24    1    a          :            (au)                 -0.508298449198
   25    1    a          :            (au)                 -0.486659823646
   26    1    a          :            (au)                 -0.471555923331
   27    1    a          :            (au)                 -0.471555761904
   28    1    a          :            (au)                 -0.446207917486
   29    1    a          :            (au)                 -0.446207130763
   30    1    b          :            (au)                -77.468205682195
   31    1    b          :            (au)                -24.841701957150
   32    1    b          :            (au)                -24.841698610770
   33    1    b          :            (au)                -24.841694025748
   34    1    b          :            (au)                -24.797497902461
   35    1    b          :            (au)                -24.797497135150
   36    1    b          :            (au)                 -6.836474532991
   37    1    b          :            (au)                 -4.975492262618
   38    1    b          :            (au)                 -4.975491913337
   39    1    b          :            (au)                 -4.972617019105
   40    1    b          :            (au)                 -1.379766211344
   41    1    b          :            (au)                 -1.316232071135
   42    1    b          :            (au)                 -1.316227609792
   43    1    b          :            (au)                 -1.277214548399
   44    1    b          :            (au)                 -1.245392473401
   45    1    b          :            (au)                 -0.741021072571
   46    1    b          :            (au)                 -0.642524953514
   47    1    b          :            (au)                 -0.642523536789
   48    1    b          :            (au)                 -0.628911138523
   49    1    b          :            (au)                 -0.571288519382
   50    1    b          :            (au)                 -0.571286086540
   51    1    b          :            (au)                 -0.549938522172
   52    1    b          :            (au)                 -0.549935623905
   53    1    b          :            (au)                 -0.524155444629
   54    1    b          :            (au)                 -0.508298452540
   55    1    b          :            (au)                 -0.486659823260
   56    1    b          :            (au)                 -0.471555923669
   57    1    b          :            (au)                 -0.471555761438
   58    1    b          :            (au)                 -0.446207917998
   59    1    b          :            (au)                 -0.446207130550
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -476.270547679643
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000020        0.000002       -0.000000
 Magnitude               :            (au)                        0.000020
                         :         (Debye)                        0.000051
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000020        0.000002       -0.000000
 Magnitude               :            (au)                        0.000020
                         :         (Debye)                        0.000051
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 34m 46s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
